SegmentEditorMaskVolume/SegmentEditorMaskVolumeLib/SegmentEditorEffect.py

```diff
@@ -44,19 +45,58 @@ def setupOptionsFrame(self):
     self.operationRadioButtons.append(self.fillOutsideButton)
     self.buttonToOperationNameMap[self.fillOutsideButton] = 'FILL_OUTSIDE'
 
+    self.binaryMaskFillButton = qt.QRadioButton("Binary mask")
```

Comment:
Could you ask a few people around you what they would think to happen if they choose the "binary mask" option? If they don't guess correctly then we might consider a simpler/more explicit name.